### ISSUE TITLE
Appium Inspector stubborn on the device name

### ISSUE THREAD

**Issue by GantMan** [appium/appium-dot-app#463]
I was having the following error:  `Failed to start an Appium session, err was: Error: The following desired capabilities are required, but were not provided: deviceName`

I googled it and found a fix here:
https://github.com/appium/appium-dot-app/issues/401

Now I'm sure I'm close to having a fix, but the issue seems to have grown, further.  Now when I select inspector I get the following!
![screen shot 2015-02-17 at 3 15 20 pm](https://cloud.githubusercontent.com/assets/997157/6237393/07aaea2c-b6b8-11e4-849f-ff7c8a4d6e8d.png)

As you can see, my first requested device (via the drop down) is `iPhone 5s (21BDED1B-88C0-4D5B-9F84-3768161001EA) (8.1 Simulator)`, which is not matching `iPhone 5s (8.1 Simulator) [21BDED1B-88C0-4D5B-9F84-3768161001EA]` because of the placement of "(8.1 Simulator)".  I can edit the drop down box, but seems to ALWAYS append "(8.1 Simulator)", causing the match to fail.   Suggestions on how to get the inspector open??

**Comment by penguinho** [comment]
I agree with this. @imurchie @jlipps Is there a way to make the server smarter or more lenient on these like it used to be.

**Comment by penguinho** [comment]
The app pulls them from `xcrun simctl list device`

**Comment by GantMan** [comment]
I'm seeing progress by just putting `iPhone 5s` in the force device.

**Comment by fangmobile** [comment]
typing name works for me ONLY IF I launch the same simulator in a lower version of Appium.app first then close that one.

**Comment by fangmobile** [comment]
I am getting the same thing in 1.3.5.
Typing in name works ONLY IF I launch the same simulator with a lower version (1.3.1) first then close it.
``` bash
\Launching Appium with command: '/Applications/Appium.app/Contents/Resources/node/bin/node' lib/server/main.js --command-timeout "7200" --pre-launch --platform-version "8.1" --platform-name "iOS" --app "/Users/jfang1/Desktop/appium/PlayerIOSDemo.app" --full-reset --device-name "iPad Air (95EA5DBB-3ADC-48F9-87AD-8732F79B2AAD)"

info: [debug] Starting Appium in pre-launch mode

info: Pre-launching app
info: [debug] Using local app from command line: /Users/jfang1/Desktop/appium/PlayerIOSDemo.app
info: [debug] Creating new appium session 5cbf1283-3d01-47e3-b4f8-7ea81927c182
info: [debug] Removing any remaining instruments sockets
info: [debug] Cleaned up instruments socket /tmp/instruments_sock
info: [debug] Setting Xcode folder

info: [debug] Setting Xcode version

info: [debug] Setting iOS SDK Version

info: [debug] Getting sdk version from xcrun with a timeout

info: [debug] iOS SDK Version set to 8.1

info: [debug] Checking whether instruments supports our device string
info: [debug] Getting list of devices instruments supports

info: [debug] Instruments is at: /Applications/Xcode.app/Contents/Developer/usr/bin/instruments

info: [debug] Getting device string from opts: {"forceIphone":false,"forceIpad":false,"xcodeVersion":"6.1.1","iOSSDKVersion":"8.1","deviceName":"iPad Air (95EA5DBB-3ADC-48F9-87AD-8732F79B2AAD)","platformVersion":"8.1"}

info: [debug] fixDevice is on
info: [debug] Final device string is: 'iPad Air (95EA5DBB-3ADC-48F9-87AD-8732F79B2AAD) (8.1 Simulator)'

info: [debug] Cleaning up appium session

error: Could not find a device to launch. You requested 'iPad Air (95EA5DBB-3ADC-48F9-87AD-8732F79B2AAD) (8.1 Simulator)', but the available devices were: ["iPad 2 (8.1 Simulator) [165412F2-9E9A-4D29-B627-29B26F75E643]","iPad Air (8.1 Simulator) [95EA5DBB-3ADC-48F9-87AD-8732F79B2AAD]","iPad Retina (8.1 Simulator) [EB97F206-6C6F-40EC-A9E7-BB264759A54D]","iPhone 4s (8.1 Simulator) [26273000-2E4E-47DA-89DD-AE058B3D0301]","iPhone 6 (8.1 Simulator) [9E279D46-A986-4721-A3F4-EC61580607E2]","iPhone 6 Plus (8.1 Simulator) [739A23F8-2DD8-438C-AD22-CE8C9F78F551]"]
```

**Comment by penguinho** [comment]
A strict caps capability now exists in the server for this and I've added it to general settings. This should be enough to make this better now.
